doc/examples/features_detection/plot_orb.py

```diff
@@ -49,8 +49,17 @@
 
 plot_matches(ax[0], img1, img2, keypoints1, keypoints2, matches12)
 ax[0].axis('off')
+ax[0].set_title("Original Image VS Transformed Image")
```

Comment:
I would instead draw the attention to the detected features, like "ORB features detected in original and transformed image".